src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -860,8 +861,14 @@ func (s *service) Write(tctx thrift.Context, req *rpc.WriteRequest) error {
 	}
 
 	if err = s.db.Write(
-		ctx, s.pools.id.GetStringID(ctx, req.NameSpace), s.pools.id.GetStringID(ctx, req.ID),
-		xtime.FromNormalizedTime(dp.Timestamp, d), dp.Value, unit, dp.Annotation,
+		ctx,
+		s.pools.id.GetStringID(ctx, req.NameSpace),
+		s.pools.id.GetStringID(ctx, req.ID),
+		xtime.FromNormalizedTime(dp.Timestamp, d),
+		dp.Value,
+		unit,
+		dp.Annotation,
+		nil,
```

Comment:
pass down default writeOpts (instead of nil).

Reply:
@justinjc ,  you can leave it as is (passing nil). I am going to construct writeOpt and pass it down

@richardartoul I will make sure namespace do not construct new writeopts unless schema has changed.

Reply:
I've removed write options from this diff since this logic doesn't rely on that. We can discuss/work on that separately in another diff.